### PR TITLE
One SDK for Azure Backup Vault powershell cmdlet.

### PR DESCRIPTION
**Pull request by swatimadhukar**
One SDK for Azure Backup Vault powershell cmdlet.
This also includes test cases for hyak.
Hyak spec PR : https://github.com/Azure/hydra-specs-pr/pull/1045

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@swatimadhukar__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by hovsepm** [comment]
@azuresdkci add to whitelist

**Comment by ogail** [comment]
@azuresdkci add to whitelist

File: src/ResourceManagement/AzureBackup/BackupServicesManagement/BackupServicesManagement.csproj [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      <HintPath>$(LibraryNugetPackageFolder)\Hyak.Common.1.0.2\lib\portable-net403+win+wpa81\Hyak.Common.dll</HintPath>
+    </Reference>
+    <Reference Include="Microsoft.Azure.Common">
+      <HintPath>$(LibraryNugetPackageFolder)\Microsoft.Azure.Common.2.1.0\lib\portable-net45+wp8+wpa81+win\Microsoft.Azure.Common.dll</HintPath>
+    </Reference>
+  </ItemGroup>
+  <Import Project="$(MSBuildExtensionsPath32)\Microsoft\Portable\$(TargetFrameworkVersion)\Microsoft.Portable.CSharp.targets" Condition=" '$(LibraryFxTarget)' == 'portable' " />
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" Condition=" '$(LibraryFxTarget)' != 'portable' " />
+  <Import Project="$(LibraryNugetPackageFolder)\Microsoft.Bcl.Build.1.0.14\tools\Microsoft.Bcl.Build.targets" Condition="Exists('$(LibraryNugetPackageFolder)\Microsoft.Bcl.Build.1.0.14\tools\Microsoft.Bcl.Build.targets')" />
+  <Import Project="$(SolutionDir)\.nuget\NuGet.targets" Condition="Exists('$(SolutionDir)\.nuget\NuGet.targets')" />
```
Comment (yugangw-msft) [inline src/ResourceManagement/AzureBackup/BackupServicesManagement/BackupServicesManagement.csproj]: please remove this and the ".nuget\nuget.targets"

File: src/ResourceManagement/AzureBackup/BackupServicesManagement/BackupServicesManagement.csproj [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
```
Comment (yugangw-msft) [inline src/ResourceManagement/AzureBackup/BackupServicesManagement/BackupServicesManagement.csproj]: please also remove this whole thing

File: src/ResourceManagement/AzureBackup/BackupServices.Tests/ScenarioTests/VaultTests.cs [hunk 3]
```diff
@@ -0,0 +1,358 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+using BackupServices.Tests.Helpers;
+using Microsoft.Azure.Management.BackupServices;
+using Microsoft.Azure.Management.BackupServices.Models;
+using Microsoft.Azure.Test;
+using System;
```
Comment (yugangw-msft) [inline src/ResourceManagement/AzureBackup/BackupServices.Tests/ScenarioTests/VaultTests.cs]: please order the namespace import, which gets the system ones at the top

**Pull request merged**